apps/condo/domains/notification/constants/constants.js

```diff
@@ -396,6 +399,15 @@ const MESSAGE_META = {
             messageBatchId: { required: false },
         },
     },
+    [VOIP_INCOMING_CALL_MESSAGE_TYPE]: {
+        dv: { required: true },
+        title: { required: false },
+        body: { required: true },
```

Comment:
What's the type of body? Are we needed to know it?

Reply:
both body and title are text.

Reply:
Does it checks anywhere? So I can be sure about it, since its not TS

Reply:
unfortunately there is no type checking in notifications payload validation. :(

Reply:
all push payload fields are forcedly converted to strings